kratos/benchmarks/model_part_benchmark.cpp

```diff
@@ -239,9 +239,12 @@ static void BM_ModelPartAddNodesToSubSubFromId1(benchmark::State& state) {
             existing_node_ids.reserve(input_size / 2 + 1);
         }
 
+        for (IndexType i = 0; i < input_size; ++i) {
+            nodes.insert(nodes.end(), Kratos::make_intrusive<NodeType>(i + 1, 0.0, 0.0, 0.0));
```

Comment:
It is `O(1)` complexity, because here, it is guranteed to have a valid hint hence the PVS will use push_back internally. Anyways, this does not affect the benchmark since this was done outside the timings of the benchmark.

Reply:
ok, that is just a benchmark